addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1492,6 +1493,183 @@ options.registry.gallery = options.Class.extend({
     },
 });
 
+options.registry.countdown = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        this.$el.on('click', '.toggle-edit-message', this._onToggleEndMessageClick.bind(this));
+        this.endTimeValue = parseInt(this.$target[0].dataset.endTime);
+        this.redirectUrlValue = this.$target[0].dataset.redirectUrl;
+        wUtils.autocompleteWithPages(this, this.$el.find('[data-redirect-url] > input'));
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy: function () {
+        this.$target.find('.s_countdown_end_message').addClass('d-none');
+        this.$target.find('.s_countdown_canvas_wrapper').removeClass('d-none');
+        this.$target.removeClass('show-end-message');
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Changes the countdown action at zero.
+     *
+     * @see this.selectClass for parameters
+     */
+    endAction: function (previewMode, value, $opt) {
+        this.$target[0].dataset.endAction = value;
+        if (value === 'message') {
+            if (!this.$target.find('.s_countdown_end_message').length) {
+                const message = this.endMessage || qweb.render('website.s_countdown.end_message');
+                this.$target.find('.container').append(message);
+            }
+        } else {
+            const $message = this.$target.find('.s_countdown_end_message').detach();
+            this.endMessage = $message.lenght ? $message[0].outerHTML : this.endMessage || null;
```

Comment:
That condition is always false... :smile:

Reply:
F**k you :cry: you did not even gave me a hint

Reply:
I will write it 10 times on the blackboard.
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**
- Leng**th**